docker/validator-dynamic/docker-run-dynamic.sh

```diff
@@ -42,7 +42,7 @@ fi
     ${params[@]}
 
 
-if [ -n "${CFG_FULLNODE_SEED}" -a "${CFG_ENABLE_MGMT_TOOL}" = false ]; then # We have a full node seed, add fullnode network
+if [[ -n "${CFG_FULLNODE_SEED}" && ("${CFG_ENABLE_MGMT_TOOL}" = false || -z "${CFG_ENABLE_MGMT_TOOL}") ]]; then
```

Comment:
nit - add a comment that a fullnode network is created iff its seed is defined and management tool is not enabled.

Reply:
Sounds good!